api/management/commands/loaddata.py

```diff
@@ -0,0 +1,17 @@
+import os
+import csv
+from django.core.management.base import BaseCommand
+from api.models import Disease
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        path = os.path.join('data', 'diseases.tsv')
+        with open(path, 'r') as diseases_file:
```

Comment:
Read mode is the default so no need to specify it. However, if you think this makes the code more readable feel free to leave it.